README.md

```diff
@@ -4,3 +4,4 @@ Assetto Corsa Competizione UDP broadcast API wrapper
 # To Do
 1. Improve documentation via docstrings for each method
 1. Add unit tests
+1. Publish as `pip` package
```

Comment:
I wonder why Github thinks this PR added this line when it's already on `master`

Reply:
Yeah, no idea... I messed up a bit with the merging and rebasing but this is still strange